src/odemis/dataio/hdf5.py

```diff
@@ -1200,132 +1030,72 @@ def _add_image_metadata(group, image, mds):
     triggerRate, st_triggerRate = [], []
 
     for md in mds:
-        if model.MD_AR_POLE in md:
-            pp.append(md[model.MD_AR_POLE])
-            st_pp.append(ST_REPORTED)
-        else:
-            pp.append((0, 0))
-            st_pp.append(ST_INVALID)
-        if model.MD_AR_XMAX in md:
-            xm.append(md[model.MD_AR_XMAX])
-            st_xm.append(ST_REPORTED)
-        else:
-            xm.append(0)
-            st_xm.append(ST_INVALID)
-        if model.MD_AR_HOLE_DIAMETER in md:
-            hd.append(md[model.MD_AR_HOLE_DIAMETER])
-            st_hd.append(ST_REPORTED)
-        else:
-            hd.append(0)
-            st_hd.append(ST_INVALID)
-        if model.MD_AR_FOCUS_DISTANCE in md:
-            fd.append(md[model.MD_AR_FOCUS_DISTANCE])
-            st_fd.append(ST_REPORTED)
-        else:
-            fd.append(0)
-            st_fd.append(ST_INVALID)
-        if model.MD_AR_PARABOLA_F in md:
-            pf.append(md[model.MD_AR_PARABOLA_F])
-            st_pf.append(ST_REPORTED)
+        # dwell time ebeam or?? exposure time # TODO exposure time and dwell time same?
```

Comment:
Yes, you've understood correctly. In HDF5, there is no difference between exposure time and dwell time, and it's all called "integration time" (and I do understand the reasoning, as they are essentially the same, and it wouldn't make sense to have both metadata on the same data). So they are merged.

Your code looks fine. It seems to do exactly the same as before.